src/elements/image/ImageElementForm.tsx

```diff
@@ -171,13 +169,15 @@ const RoleOptionsDropdown = ({
 }) => {
   // We memoise these options to ensure that this array does not change identity
   // and re-trigger useEffect unless our additionalRoleOptions have changed.
-  const allOptions = useMemo(
-    () => [...additionalRoleOptions, thumbnailOption],
-    [additionalRoleOptions]
-  );
+  const allOptions = useMemo(() => additionalRoleOptions, [
+    additionalRoleOptions,
```

Comment:
Fair to say that `additionalRoleOptions` is now just `roleOptions`? Might be worth a rename.

Reply:
I started doing this, but `roleOptions` is also used as a local variable in the `RoleOptionsDropdown` to define the final role options actually used once adjustments have been made, so it seemed to make more sense to keep it as is.